### ISSUE TITLE
install without needing python

### ISSUE THREAD

**Issue by MyTDT-Mysoft** [emscripten-core/emscripten#14252]
hi, i would like to try emscripten again... when i tried last time it was not required to install python to use it... how do it now? (without linux subsystem/emulation either)

**Comment by sbc100** [comment]
Emscripten itself is mostly written in python (with a little bit of JS).   I don't see how what you are asking for is possible without a complete re-write. 

When did you last try emscripten?... the history of emcc.py (the main entry point) goes back 10 years (way before I was working on emscripten)

**Comment by MyTDT-Mysoft** [comment]
yes i remember emscripten used python for the compilation stuff (which is sad, but too late to change)... but last time i used... 2014? it was before that big rewrite?... and now emsdk requires python to be installed beforehand (or it came with python binary already) its just frustrating that something that exist to generate cross compatibility programs... have so little of cross compatibility... last time i used on XP (i wouldnt expect it to work on XP at this time... but at least win7 would still be sane... but i never expected for nothing properly supporting 32bit even....)

**Comment by sbc100** [comment]
I see, it sounds like your beef is mostly with emsdk then.   Perhaps future bugs should be filed there, to avoid confusion.   I misunderstood and though you were asking for an python-free version of emscripten (i.e. a full re-write).

**Comment by sbc100** [comment]
https://github.com/emscripten-core/emsdk/
